.github/workflows/build.yaml

```diff
@@ -6,36 +6,23 @@ on:
     branches:
       - develop
 jobs:
-  Build_and_run_cargo:
+  nix-shell:
     name: Build and run cargo tests
     runs-on: self-hosted
-    timeout-minutes: 20
+    timeout-minutes: 30
     container:
-      image: docker.io/mayadata/ms-buildenv:latest
-      options: --privileged -v /dev:/dev -v /bin:/host/bin -v /lib/modules:/lib/modules
+      image: docker.io/mayadata/ms-buildenv:gila
```

Comment:
Is this the correct image? (thought we'd be using the base nix image now)

Reply:
I will change it at the end yes :)